_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,49 @@
+Something old, something new...
+
+
+Even though it’s only June, our HackIllinois Summer team is already well under way in planning the event for next February! We’ve been exploring tons of new possibilities for what the future of our yearly gathering will look like, and fine tuning the things that didn’t go as well last year.
+
+
+Something Old:
+
+If you haven’t already gotten an email, we’d love to hear your feedback from HackIllinois 2018! [link feedback form here] We’re engaging some of our most notable mentors from last year to get their take of the weekend: what they liked and what they would like changed. These mentors are at the heart of our hacker experience, tons of fun to have on campus, and exceptionally talented contributors in their field. If you didn’t get to engage with them at last year’s event, we didn’t do our jobs properly.
+
+2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you wouldn’t otherwise have the opportunity to learn. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
```

Comment:
potential here to expand on the sponsor side of things since this isn't something that was touched in our old posts - otherwise feels similar to what was already written